core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/api/none.kt

```diff
@@ -0,0 +1,17 @@
+package org.jetbrains.kotlinx.dataframe.api
+
+import io.kotest.matchers.shouldBe
+import org.jetbrains.kotlinx.dataframe.ncol
+import org.jetbrains.kotlinx.dataframe.nrow
+import org.junit.Test
+
+class NoneTests : ColumnsSelectionDslTests() {
+
+    @Test
+    fun none() {
+        df.select { none() }.let {
+            it.nrow shouldBe 0
```

Comment:
Could we add a test for df.select { none() and firstName } to have 1 column selected to keep the algebra:)